client/components/map/FacilityCard.tsx

```diff
@@ -1,49 +1,208 @@
-import { Flex, Text, HStack, Button } from "@chakra-ui/react";
-import { TStateFacilities } from "app-context/SheetyContext/types";
+import { Flex, Text, HStack, Button, Box, VStack, Divider } from "@chakra-ui/react";
+import { TEmptyItem, TItemSelection, TStateFacilities } from "app-context/SheetyContext/types";
 import { Image } from "@chakra-ui/react";
+import { ChevronDownIcon, ChevronUpIcon, ExternalLinkIcon } from "@chakra-ui/icons";
 import { COLORS } from "theme";
+import React, { ReactNode, useState } from "react";
+import styles from "components/home/hideScrollbar.module.css";
+import Link from "next/link";
+import { useSheetyData } from "hooks/useSheetyData";
+import { CheckIcon } from "@chakra-ui/icons";
+import { Categories } from "api/sheety/enums";
 export const FacilityCard = ({
+	items,
 	facCardDetails,
 	facCardDistance,
-	width,
-	left,
 }: {
+	items: (TItemSelection | TEmptyItem)[];
 	facCardDetails: TStateFacilities;
 	facCardDistance: number;
-	width: string;
-	left: string;
 }) => {
+	const { getItemCategory } = useSheetyData();
+
+	const TRANSLATION_DIST = -50;
+	const [translateY, setTranslateY] = useState(0);
+	const handleMovement = () => {
+		translateY === TRANSLATION_DIST ? setTranslateY(0) : setTranslateY(TRANSLATION_DIST);
+	};

```

Comment:
nit: we could rename this to `isExpanded` and make it a boolean instead of sotring the translate value directly. 

This will make some of the code easier to read:

```
{!isExpanded && (
	<Box
```

instead of 

```
{translateY === 0 && (
	<Box
```